rasterio/_base.pyx

```diff
@@ -389,9 +389,10 @@ cdef class DatasetReader(object):
             row += self.height
         return c+a*col, f+e*row
 
-    def index(self, x, y, op=math.floor):
-        """Returns the (row, col) index of the pixel containing (x, y)."""
-        return get_index(x, y, self.affine)
+    def index(self, x, y, **kwargs):
```

Comment:
Sure thing. This was one of those 50/50 decisions where I knew I'd choose incorrectly, haha.